### ISSUE TITLE
In the QA plots can we also include the distance between the standard and the science target, not just the airmasses?

### ISSUE THREAD

**Issue by ctheissen** [pyspextool/pyspextool#156]
This would be useful to investigate issues in reductions. Sometimes the standard is pretty far away from the science target which can be problematic if the weather isn't great.

**Comment by michaelcushing** [comment]
Yes!  I will include both the angular separation and the airmass difference.  Is there is a specific QA plot you want this added to?

**Comment by michaelcushing** [comment]
Is the "Reduced Spectrum and Source Information" what happens in the batch mode?  I will definitely add them in the header.

**Comment by ctheissen** [comment]
I'm realizing it might be useful to have the time different too since sometimes a standard is observed a long time past the initial science target.

**Comment by ctheissen** [comment]
> Is the "Reduced Spectrum and Source Information" what happens in the batch mode? I will definitely add them in the header.

Yes, it is. I didn't realize that wasn't produced in the normal workflow.

**Comment by michaelcushing** [comment]
@aburgasser  You will need to update the batch code once I get these new keywords in the headers.

**Comment by aburgasser** [comment]
can add to pile

**Comment by aburgasser** [comment]
this is now fixed in the QA page and fits header
